### PR TITLE
titleize helper

### PR DESCRIPTION
**Pull request by Asherlc**
Titleizes a string

### CODE REVIEW

**Comment by martndemus** [comment]
See comments on #158, #159, #160

**Comment by Asherlc** [comment]
I believe these are all taken care of.

**Comment by Asherlc** [comment]
Merge conflicts resolved

File: addon/helpers/titleize.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { helper } from 'ember-helper';
+import titleizeLib from 'ember-composable-helpers/utils/titleize';
+
+export function titleize(params/*, hash*/) {
```
Comment (poteto) [inline addon/helpers/titleize.js]: Can just destructure:

```js
export default function titleize([str]) {
```

**Comment by martndemus** [comment]
Implemented by #194

**Pull request closed without merging**